cmd/kube-apiserver/app/options/options.go

```diff
@@ -69,8 +69,10 @@ type ServerRunOptions struct {
 	ServiceClusterIPRanges string
 	// PrimaryServiceClusterIPRange and SecondaryServiceClusterIPRange are the results
 	// of parsing ServiceClusterIPRange into actual values
+	// APIServerServiceIP is the first valid IP from PrimaryServiceClusterIPRange
```

Comment:
@pandaamanda the code doc should go right above the variable itself, can you move this down two lines?

Reply:
thanks. fix it.